src/formats/index.ts

```diff
@@ -0,0 +1,27 @@
+// Import Internal Dependencies
+import { BaseStrategyFormat } from "../strategies/types/api.js";
+
+import {
+  standardVulnerabilityMapper,
+  StandardizeKind
+} from "./standard/index.js";
+
+export function formatVulnsPayload(format: BaseStrategyFormat = "Preserve") {
```

Comment:
I wonder if we should keep `Preserve` format.

We could have only `Standard` & `OSV`. If the user wanna "preserve" then he just give `undefined` or `null` format (which would be the default), wdyt ?

Reply:
Yes could be a good idea (however depend on the implementation because I think we use that type in multiple location).

Anyway in the long terme standard should dissapear in favor of OSV (which is the real open source standard).